index.html

```diff
@@ -0,0 +1,801 @@
+                        <h3 class="font-weight-bold my-1 ">Inside the Last Bohemian Apartments of the Storied Hotel Chelsea</h3>
+                    </article>
+                    <article class="my-3 col-md-4">
+                        <div class="position-relative">
+                            <img class="article-image"
+                                src="https://d.newsweek.com/en/full/1560097/tozeur-tunisia.webp?w=397&h=265&f=69a77ef1bd7e21c060a7b7685b05a73d"
+                                alt="featured-stories">
+                            <img class="hashtag position-absolute" src="./imgs/slideshow-num-icon.png" alt="">
+                        </div>
+                        <h3 class="font-weight-bold my-1 ">Relive the 'Star Wars' Saga at These Filming Locations</h3>
+                    </article>
+                    <article class="my-3 col-md-4">
+                        <div class="position-relative">
+                            <img class="article-image"
+                                src="https://d.newsweek.com/en/full/1552749/eyjafjallajokull-iceland.webp?w=397&h=265&f=ea1ffc67b32c527ef435529cb5bd2dc6"
+                                alt="featured-stories">
+                            <img class="hashtag position-absolute" src="./imgs/slideshow-num-icon.png" alt="">
+                        </div>
+                        <h3 class="font-weight-bold my-1 ">Active Volcanoes Around the World That Could Erupt at Any Moment</h3>
+                    </article>
+                </div>
+            </section>
+            <hr>
+
+            <section class="category py-3">
+                <h3 class="category-title">U.S.</h3>
+                <div class="category-container row">
+                    <article class="col-md-4">
+                        <img class="category-main-img" src="https://d.newsweek.com/en/full/1573074/coronavirus-recoveries-us-global-cases.webp?w=397&h=265&f=0e215f9d6b052a4451c36c89b8126425"
+                            alt="featured-stories">
+                        <h3 class="pt-2 pt-md-0">Over 66,000 Have Recovered From Coronavirus, Per Johns Hopkins University</h3>
+                    </article>
+                    <article class="col-md-4">
+                        <img class="category-main-img" src="https://d.newsweek.com/en/full/1573102/super-tuesday-primary-voters.webp?w=397&h=265&f=b7119d3c9cce244963cbbb96b607d0cd"
+                            alt="featured-stories">
+                        <h3 class="pt-2 pt-md-0">Democratic Delegate Count Update: Results For Biden, Sanders and Gabbard</h3>
+                    </article>
+                    <aside class="category-aside col-md-4 row">
+                        <article class="row">
+                            <img class="category-aside-img col-lg-5 col-md-6 col-3 px-3 pl-4"
+                                src="https://d.newsweek.com/en/full/1573028/dr-anthony-fauci-covid-19-briefing.webp?w=95&h=95&f=2e43e6e9e762cda94d8010ffc0f163af"
```

Comment:
- [ ] The images on the `aside` of these sections (the ones you are not showing yet) should be a little smaller. Please check all the breakpoints and compare them to the original page.

- [ ] Check the second article's image in the mobile version: it has less width than the others but that should be the size of all the images in that screen size.